src/bin/cli/contract/mod.rs

```diff
@@ -35,10 +31,38 @@ pub struct ContractData {
     pub tokens: TokenizedAbi,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct ContractParserConfig {
+    /// The file extension that should be considered as a Sierra file.
+    pub sierra_extension: String,
```

Comment:
Perhaps we could add 
```rust
pub struct ContractParserConfig {
  #[serde(default = ".contract_class.json")]
  pub sierra_extension: String,
  
  #[serde(default)]
  pub type_aliases: HashMap<String, String>,
}
```

Reply:
When i think about it I have some doubts :)). So imo we can add this or leave it as it is

Reply:
> Perhaps we could add
> 
> ```rust
> pub struct ContractParserConfig {
>   #[serde(default = ".contract_class.json")]
>   pub sierra_extension: String,
>   
>   #[serde(default)]
>   pub type_aliases: HashMap<String, String>,
> }
> ```

Isn't the default implementation already defined below?

Reply:
Yes, you're right, I've missed that. We could then have
```rust
#[derive(Debug, Clone, Serialize, Deserialize)]
#[serde(default)] 
pub struct ContractParserConfig {
    pub sierra_extension: String,
    pub type_aliases: HashMap<String, String>,
}
```
This would allow for the fields to be unspecified when the user wants to have the default

Reply:
> Yes, you're right, I've missed that. We could then have
> 
> ```rust
> #[derive(Debug, Clone, Serialize, Deserialize)]
> #[serde(default)] 
> pub struct ContractParserConfig {
>     pub sierra_extension: String,
>     pub type_aliases: HashMap<String, String>,
> }
> ```
> 
> This would allow the fields to be left unspecified when the user wants to have the default

Yep I did that at first, but I wanted to enforce the sierra extension. But as you mention, we may prefer having a 100% default value, and having the `.contract_class.json` only in the example file? This is what you suggest?